package.json

```diff
@@ -34,11 +34,11 @@
   },
   "homepage": "https://github.com/techchange/console-to-slack#readme",
   "license": "MIT",
-  "dependencies": {
-    "axios": "^0.27.2"
-  },
   "devDependencies": {
     "eslint": "^8.20.0",
     "eslint-config-techchange": "^4.0.1"
+  },
+  "peerDependencies": {
+    "axios": "^1.7.4"
```

Comment:
In general, what are the implications of having different versions of packages across our repos? Are there scenarios where that causes problems, and scenarios where it's okay?